### PR TITLE
About us/contact us

### PR DESCRIPTION
**Pull request by Rajat-rajat**
Review this code

### CODE REVIEW

File: src/pages/AboutUs.js
Comment (rimple0) [inline src/pages/AboutUs.js:1]: Make common for className='live-counts-box'.  Take ContentSectionsheading in variable

File: src/pages/ContactUs.js
Comment (rimple0) [inline src/pages/ContactUs.js:1]: add variable for common ClassNames and make code file of 120 lines

File: src/pages/Home.js
Comment (rimple0) [inline src/pages/Home.js:1]: take variable for para and ContentSectionspara

File: src/pages/AboutUs.js
Comment (Rajat-rajat) [reply src/pages/AboutUs.js:1]: done mam

File: src/pages/ContactUs.js
Comment (Rajat-rajat) [reply src/pages/ContactUs.js:1]: done mam

File: src/pages/Home.js
Comment (Rajat-rajat) [reply src/pages/Home.js:1]: done mam

**Review by rimple0 — APPROVED** [review]
Approved

**Pull request merged**